themes/StarterTheme/templates/catalog/_partials/active_filters.tpl

```diff
@@ -0,0 +1,10 @@
+{if $activeFilters|count}
+  <section class="active_filters">
+  <h1 class="h3">{l s='Active Filters'}</h1>
```

Comment:
I'm concerned that not having `<aside><nav>~</nav></aside>` would be adding the H1 into the main body meta picked up in search. I've seen it before with left column infoboxes.